pages/booking/booking.tsx

```diff
@@ -0,0 +1,114 @@
+import { styled } from '@mui/material/styles'
+import { useState } from 'react'
+
+import Card from '@mui/material/Card'
+import CardActions from '@mui/material/CardActions'
+import CardContent from '@mui/material/CardContent'
+import Collapse from '@mui/material/Collapse'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import IconButton, { IconButtonProps } from '@mui/material/IconButton'
+import Typography from '@mui/material/Typography'
+
+interface ExpandMoreProps extends IconButtonProps {
+    expand: boolean
+}  
+
+const ExpandMore = styled((props: ExpandMoreProps) => {
+        const { expand, ...other } = props
+        return <IconButton {...other} />
+    })(({ theme, expand }) => ({
+        transform: !expand ? 'rotate(0deg)' : 'rotate(180deg)',
+        marginLeft: 'auto',
+        transition: theme.transitions.create('transform', {
+            duration: theme.transitions.duration.shortest,
+        })
+    }))
+
+function Booking(data) {
+    const [expanded, setExpanded] = useState(false)
+  
+    function handleExpandClick(i) {
+      setExpanded(expanded === i ? false : i)
+    }
+
+    const startsat = new Date(data.children.startsat)
+    const endsat = new Date(data.children.endsat)
+    const createdat = new Date(data.children.createdat)
+  
+    return (
+        <>
+            <Card sx={{ width: 'auto', height: 'auto', m: 2}} elevation={5}>
+                <CardContent>
+                    <Typography variant="h6" gutterBottom>
+                        <b>Name:</b> {data.children.name}
+                    </Typography>
+                    <Typography variant="body2">
+                        <b>Phone:</b> {data.children.phone}
+                    </Typography>
+                    <Typography variant="body2">
+                        <b>Mail:</b> {data.children.mail}
+                    </Typography>
+                    <Typography variant="body2">
+                        <b>Starts at: </b>{startsat.toLocaleString('ro-RO')}
+                        <b> Ends at: </b>{endsat.toLocaleString('ro-RO')}
+                    </Typography>
+                </CardContent>
+                <CardActions disableSpacing>
+                    <ExpandMore
+                        expand={expanded === data.children.id}
+                        onClick={() => handleExpandClick(data.children.id)}
+                        aria-expanded={expanded === data.children.id}
+                        aria-label="show more"
+                    >
+                    <ExpandMoreIcon />
+                    </ExpandMore>
+                </CardActions>
+                <Collapse in={expanded === data.children.id} timeout="auto" unmountOnExit>
+                    <CardContent>
+                        <Typography>
```

Comment:
can we have a global list of fields and just iterate through them so we don't have repetitive code?
Something like this
```js
bookingFields: [
  {
    field_id: "utm_source",
    field_name: "UTM SOURCE",
    field_value // we might need a function instead of this array. Basically we don't want to repeat code
]
```